### PR TITLE
Update RAK devices

### PR DESCRIPTION
**Pull request by xoseperez**
#### Summary
Update RAK devices

#### Changes
* Adds RAK2270 Sticker Tracker support
* Normalizes RAK uplink decoders

### CODE REVIEW

File: vendor/rakwireless/rak2270.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+# Valid values are: none, read protected and secure element.
+keySecurity: none
+
+# Product and data sheet URLs (optional)
+productURL: https://store.rakwireless.com/products/rak2270-rak-sticker-tracker
+dataSheetURL: https://docs.rakwireless.com/Product-Categories/WisNode/RAK2270
+
+# Photos
+photos:
+  main: rak2270.png
```
Comment (Jaime-Trinidad) [inline vendor/rakwireless/rak2270.yaml:54]: Can you check the image it is causing an error `bin/validate-image ./vendor/rakwireless/rak2270.png
2023/09/04 15:20:16 png.Decode failed: Unknown file format` and run again make validate commands

**Comment by Jaime-Trinidad** [comment]
Hi @xoseperez thanks for updating, can you check the comment and rebase so I can merge it. Thanks

**Review by Jaime-Trinidad — APPROVED** [review]

**Comment by Jaime-Trinidad** [comment]
Hi @xoseperez any update here?

**Comment by xoseperez** [comment]
Sorry, I don't really know what was wrong with that PNG.... changed it to JPG and it validates.

**Comment by Jaime-Trinidad** [comment]
Thanks @xoseperez looks that the fixes worked, now please just run `make validate fmt`

**Comment by xoseperez** [comment]
Done

**Pull request merged**